R/check_ebp_arguments.R

```diff
@@ -132,13 +142,13 @@ ebp_check2 <- function(threshold, transformation, interval, MSE, boot_type, B,
                      threshold might not included in the indicator. For help
                      see Example 2 in help(ebp)."))
       } else if (inherits(custom_indicator[[i]], "function") &&
-        !all(names(formals(custom_indicator[[i]])) ==
-          c("y", "threshold"))) {
+        !all(names(formals(custom_indicator[[i]])) %in%
+          c("y", "pop_weights", "threshold"))) {
         stop(strwrap(prefix = " ", initial = "",
                      "Functions for custom indicators need to have exactly the
-                     following two arguments: y, threshold; even though a
-                     threshold might not included in the indicator. For help
-                     see Example 2 in help(ebp)."))
+                     following two arguments: y, weights, threshold; even though
```

Comment:
It would be three arguments, but I thought that is not needed anymore. But maybe I do not remember correctly. Also it would be pop_weights/population_weights